zuul.d/jobs.yaml

```diff
@@ -421,6 +421,29 @@
     vars:
       ansible_test_python: 3.6
 
+- job:
+    name: ansible-test-network-integration-ansible-utils
+    abstract: true
+    dependencies:
+      - name: build-ansible-collection
+        soft: true
+    parent: ansible-network-eos-appliance
```

Comment:
we can drop this, perhaps the parent should be

ansible-test-integration-base, and we can move common bits into it